### PR TITLE
Pfappserver chosen improvements

### PR DESCRIPTION
**Pull request by satkunas**
# Description
- Improves allowed lookup search by preventing duplicates (`oses`).
- Hides duplicate chosen placeholder on allowed lookups.
- Remove exceptions on unhandled meta.

# Impacts
None

# Delete branch after merge
YES 

# Checklist
- [n/a] Document the feature
- [n/a] Add unit tests
- [n/a] Add acceptance tests (TestLink)

### CODE REVIEW

File: html/pfappserver/root/static.alt/src/globals/configuration/pfConfiguration.js [hunk 1]
```diff
@@ -233,13 +232,13 @@ export const pfConfigurationValidatorsFromMeta = (meta = {}, key = null, fieldNa
               case 'array': // ignore
               case 'string': // ignore
                 break
-              default: // TODO: remove post-devel
-                throw new Error(`Unhandled meta type: ${meta[key].type}`)
+              default:
+                console.error(`Unhandled meta type: ${meta[key].type}`)
```
Comment (cgx) [inline html/pfappserver/root/static.alt/src/globals/configuration/pfConfiguration.js:237]: Prefix `console` statement with `// eslint-disable-next-line`.

File: html/pfappserver/root/static.alt/src/globals/configuration/pfConfiguration.js [hunk 2]
```diff
@@ -233,13 +232,13 @@ export const pfConfigurationValidatorsFromMeta = (meta = {}, key = null, fieldNa
...
                 // break
             }
             break
-          default: // TODO: remove post-devel
-            throw new Error(`Unhandled meta: ${property}`)
+          default:
+            console.error(`Unhandled meta: ${property}`)
```
Comment (cgx) [inline html/pfappserver/root/static.alt/src/globals/configuration/pfConfiguration.js:243]: Prefix `console` statement with `// eslint-disable-next-line`.

**Review by cgx — CHANGES_REQUESTED** [review]

**Comment by satkunas** [comment]
@cgx done

**Comment by cgx** [comment]
Some visual glitches I noticed.

The placeholder seems misaligned in a single-value instance:

chosen-misaligned-label

The border is partially hidden in a multi-value instance:

chosen-multiselect-border

**Comment by satkunas** [comment]
> Some visual glitches I noticed.

@cgx I am not able to reproduce either of these issues. Could this be browser related?

**Comment by cgx** [comment]
@satkunas Maybe it's because I'm running [Edge Canary](https://www.microsoftedgeinsider.com/) on macOS .. 🎲

**Comment by julsemaan** [comment]
@satkunas, I run Netscape on my Palm Pilot and the admin appears in black and white. Can you help ?

Side note: I'm color-blind

**Review by cgx — APPROVED** [review]

**Pull request merged**